### PR TITLE
21 as a logged in user i need to list all the users of the system

### LINKED ISSUE #21: As a logged in user, I need to list all the users of the system

### PR DESCRIPTION
**Pull request by sagarbhandari-0522**
An API for listing all the users of the system.

### CODE REVIEW

**Comment by shashankdhungel** [comment]
- [x] Code readability and comments
- [x] Implements required functionality
- [x] Unit tests provided and passing
- [x] Integration tests provided and passing
- [x] Authentication and authorization handled correctly
- [x] Dependencies are up-to-date
- [x] README/changelog updated if necessary
- [x] Build passes
- [x] Consistent with existing codebase
- [x] Previous review feedback addressed
- [x] Required approvals obtained

File: users/serializers.py [hunk 1]
```diff
@@ -32,3 +32,9 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
 
         return user
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'email', 'first_name', 'last_name']
```
Comment (shashankdhungel) [inline users/serializers.py]: Hey mate, we don't need first_name, last_name in the listing of users. We only need to send username and respective email through API. Why to make API bulgy?

**Review by shashankdhungel — COMMENTED** [review]
Please remove first_name and last_name from API

File: users/serializers.py (hunk 1 above)
Comment (sagarbhandari-0522) [reply users/serializers.py]: Okay, I will make this changes and resubmit new pr so that you can review it again.

File: users/serializers.py [hunk 2]
```diff
@@ -32,3 +32,9 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
 
         return user
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'email']
```
Comment (shashankdhungel) [inline users/serializers.py:40]: Okay, it is right now. Thank you for acknowledging my comment and improving the code.

**Review by shashankdhungel — APPROVED** [review]
All the test and build are successfully carried out. I will approve your code and you can merge it. And close the issues back.

**Pull request merged**